src/components/tasks/card/index.tsx

```diff
@@ -21,40 +26,61 @@ const Card: FC<Props> = ({
   const statusNotOverDueList = [COMPLETED, VERIFIED, AVAILABLE];
   const cardDetails = content;
   const [assigneeProfilePic, setAssigneeProfilePic] = useState(
-    `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`,
+    `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`
   );
-  const contributorImageOnError = () => setAssigneeProfilePic('/dummyProfile.png');
+  const contributorImageOnError = () =>
+    setAssigneeProfilePic("/dummyProfile.png");
 
   const localStartedOn = new Date(parseInt(cardDetails.startedOn, 10) * 1000);
   const fromNowStartedOn = moment(localStartedOn).fromNow();
 
   const localEndsOn = new Date(parseInt(cardDetails.endsOn, 10) * 1000);
   const fromNowEndsOn = moment(localEndsOn).fromNow();
-  const statusFontColor = !statusRedList.includes(cardDetails.status) ? '#00a337' : '#f83535';
-  const iconHeight = '25px';
-  const iconWidth = '25px';
+  const statusFontColor = !statusRedList.includes(cardDetails.status)
+    ? "#00a337"
+    : "#f83535";
+  const iconHeight = "25px";
+  const iconWidth = "25px";
 
   const cardClassNames = [classNames.card];
 
+  const changeToDate = new Date(localEndsOn);
+  const dates = JSON.stringify(changeToDate).slice(1, 11);
+
+  const [dateTime, setdateTime] = useState(dates);
+
   function isTaskOverdue() {
     const currentDate = new Date();
     const timeLeft = localEndsOn.valueOf() - currentDate.valueOf();
     return !statusNotOverDueList.includes(cardDetails.status) && timeLeft <= 0;
   }
 
   function stripHtml(html: string) {
-    const tmp = document.createElement('DIV');
+    const tmp = document.createElement("DIV");
     tmp.innerHTML = html;
-    return tmp.textContent || tmp.innerText || '';
+    return tmp.textContent || tmp.innerText || "";
   }
 
   function handleChange(event: any, changedProperty: keyof typeof cardDetails) {
-    if (event.key === 'Enter') {
+    setdateTime(event.target.value);
+
+    const inputDate = event.target.value;
+    const DateToString = new Date(inputDate);
```

Comment:
Typo `dateToString`

Reply:
I've done these task.
And I have updated PR again.
Please, have a look